airsonic-main/src/main/resources/liquibase/11.0/cue-support.xml

```diff
@@ -0,0 +1,59 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="startposition_001" author="yetangitu">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="media_file" columnName="start_position" />
+            </not>
+        </preConditions>
+        <addColumn tableName="media_file">
+            <column name="start_position" afterColumn="type" type="double" defaultValueNumeric="-1.0">
```

Comment:
That's be rather confusing given the presence of an index with that name. What is the problem with the current name?

An alternative would be `cue_point` but `start_position` is more descriptive.

Reply:
What we're trying to do is connect start_position with index_path since both these columns are linked. indexed_start_position? start_indexed_position?